common/data_refinery_common/migrations/0014_organism_experiments.py

```diff
@@ -15,4 +15,4 @@ class Migration(migrations.Migration):
             name='experiments',
             field=models.ManyToManyField(through='data_refinery_common.ExperimentOrganismAssociation', to='data_refinery_common.Experiment'),
         ),
-    ]
+]
```

Comment:
Is this change on purpose and/or meaningful? I probably wouldn't worry about it but you know python and its whitespace.

Reply:
Just my editor trying to be helpful